.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
```

Comment:
Would this be like our label tag on the issue?

Reply:
Yep. You have the option to specify labels by default. I figured these would most likely be added by me after grooming them.